metrics/error-report/Bugsnag.js

```diff
@@ -21,6 +22,15 @@ const SDK_FILE_NAMES = [
   "rudder-analytics.js",
 ];
 
+// list of not allowed errors
+const errorsToIgnore = [
```

Comment:
This is not a sustainable way to filter errors. If there is a chance for these kinds of errors to originate at the SDK, we'll lose them.

Reply:
Here we will keep the errors that will never be generated by the SDK.

Reply:
There is no way we can be certain of that just by looking at the error messages. Moreover, the error messages can transform into a new format over time.